### PR TITLE
Unique sandbox name

### PR DESCRIPTION
**Pull request by cyril-sf**

### CODE REVIEW

**Comment by cyril-sf** [comment]
@hjdivad Good for review

File: test/iframe_sandbox_tests.js [hunk 1]
```diff
@@ -25,19 +25,26 @@ module('iframe Sandboxes', {
...
 
-  sandbox.start();
+  sandbox2 = createSandbox({
+    url: 'fixtures/index.js'
+  });
+  sandbox2.start();
 
-  equal( sandbox.el.name, 'fixtures/index.js', 'The iframe has a name' );
+  ok( sandbox1.el.name !== sandbox2.el.name, 'The iframes have a unique name');
+  ok( sandbox1.el.name.match('fixtures/index.js').length, 'The iframe name contains the url' );
```
Comment (hjdivad) [inline test/iframe_sandbox_tests.js]: This is a weird way to write this test.  How about using `Regexp#test` instead?

``` js
ok( /fixtures\/index.js/.test(sandbox1.el.name), 'The iframe name contains the url' );
```

File: test/webworker_sandbox_tests.js [hunk 2]
```diff
@@ -1,22 +1,48 @@
...
+    sandbox1.start();
+
+    sandbox2 = Oasis.createSandbox({
+      adapter: webworkerAdapter,
+      url: 'fixtures/index.js'
+    });
+    sandbox2.start();
+
+    ok( sandbox1.worker.name !== sandbox2.worker.name, 'The workers have a unique name');
+    ok( sandbox1.worker.name.match('fixtures/index.js').length, 'The worker name contains the url' );
```
Comment (hjdivad) [inline test/webworker_sandbox_tests.js]: same here, prefer `Regexp#test` over `String#match`?

File: test/iframe_sandbox_tests.js [hunk 3]
```diff
@@ -25,19 +25,26 @@ module('iframe Sandboxes', {
   }
 });
 
-test("The iframes are named to improve debugging", function() {
+test("The iframes are uniquely named to improve debugging", function() {
```
Comment (hjdivad) [inline test/iframe_sandbox_tests.js]: We should mention in a comment or change this message to indicate that it's also to handle the back button in safari.  Right now it's not clear from the tests that it's _way_ more important to do this for iframes than web workers.

**Comment by hjdivad** [comment]
@Cyril-sf couple very minor points, otherwise looks good.

File: test/iframe_sandbox_tests.js (hunk 1 above)
Comment (cyril-sf) [inline test/iframe_sandbox_tests.js]: @hjdivad indeed

**Pull request merged**